### ISSUE TITLE
Dropwizard dependency has vulnerability 

### ISSUE THREAD

**Issue by mwedgwood** [xvik/dropwizard-guicey#86]
dropwizard-core-2.0.0 has the following vulnerability:

CVE-2020-5245

"Dropwizard-Validation before 1.3.19, and 2.0.2 may allow arbitrary code execution on the host system, with the privileges of the Dropwizard service account, by injecting arbitrary Java Expression Language expressions when using the self-validating feature. The issue has been fixed in dropwizard-validation 1.3.19 and 2.0.2."

Can you upgrade to https://mvnrepository.com/artifact/io.dropwizard/dropwizard-core/2.0.2?

Should be fairly trivial.

**Comment by xvik** [comment]
Ok, I give up :) (its not the first request, but I was rejecting them). I'll do release today or tomorrow.

In general, I don't want to release guicey on each dropwizard update simply because it takes much time to release everything (~1h). I encourage everyone to use BOM, which makes dropwizard updates trivial:

```groovy
plugins {
    ...
    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
}

dependencyManagement {
        imports {
            mavenBom 'ru.vyarus.guicey:guicey-bom:5.0.0-0'
            mavenBom 'io.dropwizard:dropwizard-bom:2.0.2'
            mavenBom 'io.dropwizard:dropwizard-dependencies:2.0.2'
        }
}
```

Or use guicey itself as BOM (if extensions are not required):

```groovy
plugins {
    ...
    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
}

dependencyManagement {
        imports {
            mavenBom 'ru.vyarus:dropwizard-guicey:5.0.0'
            mavenBom 'io.dropwizard:dropwizard-bom:2.0.2'
            mavenBom 'io.dropwizard:dropwizard-dependencies:2.0.2'
        }
}
```

As I receive sync-release requests quite often, I decided to set a new goal on [patreon](https://www.patreon.com/guicey): as soon as I get 10 patrons, I'll start releasing guicey for each new dropwizard release. 

Now I would have a legit way to reject similar requests in the future and the community can force me to do it :)

**Comment by mwedgwood** [comment]
"In general, I don't want to release guicey on each dropwizard update simply because it takes much time to release everything (~1h)"

Totally understand and much appreciated! This is a critical dependency for us and we have policies in place that won't allow use of 3rd party libs with CVSS score > 7.0. This particular one clocks in at at 9 :)

**Comment by xvik** [comment]
> Totally understand and much appreciated! This is a critical dependency for us and we have policies in place that won't allow use of 3rd party libs with CVSS score > 7.0. This particular one clocks in at at 9 :)

Good enough reason for the release. 

Released [5.0.1](https://github.com/xvik/dropwizard-guicey/releases/tag/5.0.1), ext [5.0.1-1](https://github.com/xvik/dropwizard-guicey-ext/releases/tag/5.0.1-1)

Updated [docs](http://xvik.github.io/dropwizard-guicey/5.0.1/)
